pkg/platform/runtime/runtime.go

```diff
@@ -183,27 +188,21 @@ func (r *Runtime) recordCompletion(err error) {
 		}
 	} else {
 		action = anaConsts.ActRuntimeSuccess
-		r.recordUsage()
+		r.recordUsage(dims)
 	}
 
 	r.analytics.EventWithLabel(anaConsts.CatRuntime, action, anaConsts.LblRtFailEnv, &dimensions.Values{
 		CommitID: p.StrP(r.target.CommitUUID().String()),
 	})
 }
 
-func (r *Runtime) recordUsage() {
+func (r *Runtime) recordUsage(dims *dimensions.Values) {
 	if !r.target.Trigger().IndicatesUsage() {
 		logging.Debug("Not recording usage as %s is not a usage trigger", r.target.Trigger().String())
 		return
 	}
 
-	dims := &dimensions.Values{
-		Trigger:          p.StrP(r.target.Trigger().String()),
-		Headless:         p.StrP(strconv.FormatBool(r.target.Headless())),
-		CommitID:         p.StrP(r.target.CommitUUID().String()),
-		ProjectNameSpace: p.StrP(project.NewNamespace(r.target.Owner(), r.target.Name(), r.target.CommitUUID().String()).String()),
-		InstanceID:       p.StrP(instanceid.ID()),
-	}
+	dims.Headless = p.StrP(strconv.FormatBool(r.target.Headless()))
```

Comment:
This feels like it should be part of `usageDims()`. If this was just to respect the dimensions section in the documentation then we can just update that to also mention headless.